compiler/src/iree/compiler/ExternalInterfaces/UtilExternalModels.cpp

```diff
@@ -521,9 +524,8 @@ void registerUtilExternalModels(DialectRegistry &registry) {
             tensor::ExtractSliceOp>::registerOpInterface(context);
         // Cases of trivial pack/unpack should be handled as canonicalizations
         // before we get here, thus we're safe to always hoist.
```

Comment:
Nit: this comment should be moved up to the linalg section since the pack/unpack ops are no longer in tensor.

Reply:
I think even the name `HoistableLinalgOpInterfaceHelper` needs to be updated as well because it is no longer linalg specific.